cargo-pgrx/src/command/install.rs

```diff
@@ -418,12 +418,12 @@ pub(crate) fn find_library_file(
     manifest_path: &Path,
     build_command_messages: &Vec<CargoMessage>,
 ) -> eyre::Result<PathBuf> {
-    use std::env::consts::{DLL_EXTENSION, DLL_SUFFIX};
+    use std::env::consts::DLL_EXTENSION;
 
     // cargo sometimes decides to change whether targets are kebab-case or snake_case in metadata,
     // so normalize away the difference
-    let target_name = manifest.target_name()?.replace('-', "_");
```

Comment:
What's the reason for removing line 425 but not also updating the comment?  Now I don't know what the intent is supposed to be.

Reply:
Removed it as it's useless now according to the selected logic.